scripts/prepdocs.ps1

```diff
@@ -61,12 +61,17 @@ if ($env:AZURE_TENANT_ID) {
   $tenantArg = "--tenantid $env:AZURE_TENANT_ID"
 }
 
+if ($env:USE_FEATURE_INT_VECTORIZATION) {
+  $integratedVectorizationArg = "--useintvectorization $env:USE_FEATURE_INT_VECTORIZATION"
+}
+
 $cwd = (Get-Location)
 $dataArg = "`"$cwd/data/*`""
 
 $argumentList = "./scripts/prepdocs.py $dataArg --verbose " + `
-"--storageaccount $env:AZURE_STORAGE_ACCOUNT --container $env:AZURE_STORAGE_CONTAINER " + `
-"--searchservice $env:AZURE_SEARCH_SERVICE --index $env:AZURE_SEARCH_INDEX " + `
+"--subscriptionid $env:AZURE_SUBSCRIPTION_ID " + `
+"--storageaccount $env:AZURE_STORAGE_ACCOUNT --container $env:AZURE_STORAGE_CONTAINER --storageresourcegroup $env:AZURE_STORAGE_RESOURCE_GROUP " + `
+"--searchservice $env:AZURE_SEARCH_SERVICE --index $env:AZURE_SEARCH_INDEX --searchserviceassignedid $env:AZURE_SEARCH_SERVICE_ASSIGNED_USERID " + `
```

Comment:
I see the .sh script is missing --searchserviceassignedid

Reply:
I think this is still an issue? But it still seems to work? How is that possible? (I'm using sh)